packages/frontend/components/composed/issue/PrevNextIssue/PrevNextIssue.styles.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+import { Button } from "components/atomic";
+import { aButton } from "theme/mixins/appearance";
+
+export const Inner = styled.div`
+  padding-block: var(--container-padding-lg);
+`;
+
+export const FixedWidthButton = styled(Button)`
+  ${aButton("primary", "lg")}
+  width: ${pxToRem(180)};
+  justify-content: center;
+
+  & > svg {
+    flex-grow: 0;
+  }
+`;
```

Comment:
@jen-castiron I created this styled component to ensure that the Prev and Next buttons are the same size. Should we add this as a pattern to `Button` or just leave here for now?

Reply:
Let's leave this here for now - we might run into issues with mobile devices, and we can check when we work on the mobile styling